test_conformance/SVM/test_enqueue_api.cpp

```diff
@@ -177,11 +177,16 @@ int test_svm_enqueue_api(cl_device_id deviceID, cl_context c, cl_command_queue q
       error = clEnqueueSVMMemcpy(queue, CL_FALSE, dst_ptr, src_ptr, elementNum * typeSizes[i], 2, &eventMemFillList[0], &eventMemcpy);
       test_error(error, "clEnqueueSVMMemcpy failed");
 
-      clEventWrapper eventMap;
-      error = clEnqueueSVMMap(queue, CL_FALSE, CL_MAP_READ | CL_MAP_WRITE, dstBuffer, elementNum * typeSizes[i], 1, &eventMemcpy, &eventMap);
-      test_error(error, "clEnqueueSVMMap failed");
+      //coarse grain only supported. Synchronization required using map
+      clEventWrapper eventMap[2];
 
-      error = clWaitForEvents(1, &eventMap);
+      error = clEnqueueSVMMap(queue, CL_FALSE, CL_MAP_READ | CL_MAP_WRITE, srcBuffer, elementNum * typeSizes[i], 1, &eventMemcpy, &eventMap[0]);
```

Comment:
`CL_MAP_WRITE` is not necessary though it's harmless. You could also introduce a variable for `elementNum * typeSizes[i]`, it's used in quite a few places.